src/tests/uat/platform_devices_uat.py

```diff
@@ -53,6 +53,14 @@ def search_devices_api_call():
     return requests.post(search_url, json=data, headers=HEADERS)
 
 
+def normalize_results(result):
+    """Helper function to sort dictionary values"""
+    for index in range(0, len(result)):
```

Comment:
This works, but you can iterate through data structures like so:
for index in (results):
   ...